src/files-footer-detail.tsx

```diff
@@ -118,7 +123,46 @@ export const FilesFooterDetail = ({
                     {timeformat.distanceToNow(selectedFile.mtime * 1000)}
                 </Text>
             </Tooltip>}
+            {userGroup}
             {permsPopover}
         </div>
     );
 };
+
+const UserGroupPopover = ({ file }: { file: FileInfo }) => (
+    <Popover
+        id="files-footer-usergroup-popover"
+        hasAutoWidth
+        bodyContent={
+            <DescriptionList
+                isHorizontal
+                isCompact
+            >
+                <DescriptionListGroup key="user">
+                    <DescriptionListTerm>
+                        User
```

Comment:
This needs to be translated

```suggestion
{_("User")}
```